config/identity/config.yaml

```diff
@@ -104,6 +104,12 @@ oidc-issuers:
     ci-provider: *github-type
     contact: tac@sigstore.dev
     description: "GitHub Actions OIDC auth"
+  https://sigstore.verify.ibm.com/oauth2:
+    issuer-url: https://sigstore.verify.ibm.com/oauth2
+    client-id: sigstore
+    type: email
+    contact: stefanb@us.ibm.com
```

Comment:
is there a way you can use a group alias instead (in case you leave IBM at some point)?

Reply:
Updated.